actionpack/test/dispatch/request_test.rb

```diff
@@ -4,10 +4,6 @@
 
 class BaseRequestTest < ActiveSupport::TestCase
   def setup
-    @env = {
-      :ip_spoofing_check => true,
```

Comment:
I thought this was a behavior change at first but I see why its not: `ip_spoofing_check` defaults to `true` in `stub_request` if the key isn't present. So this makes sense to me :+1: